easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '3.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('libjpeg-turbo', '1.4.2','-NASM-2.12.01'),
+    ('libpng', '1.6.21'),
+    ('zlib', '1.2.8'),
+    ('LibTIFF', '4.0.6'),
+    ('freetype', '2.6.3'),
```

Comment:
@FokkeDijkstra looks like you'll need to stick to freetype 2.6.2 (as in the existing Pillow 3.2.0 easyconfig):

```
Conflict found for dependencies of scikit-image-0.12.3-foss-2016a-Python-2.7.11: freetype-2.6.2-foss-2016a vs freetype-2.6.3-foss-2016a 
	freetype-2.6.2-foss-2016a as dep of: FLTK-1.3.3-foss-2016a, GL2PS-1.3.9-foss-2016a, GObject-Introspection-1.47.1-foss-2016a, GraphicsMagick-1.3.23-foss-2016a, HTSeq-0.6.1p1-foss-2016a-Python-2.7.11, HarfBuzz-1.1.3-foss-2016a, Mesa-11.1.2-foss-2016a, PLplot-5.11.1-foss-2016a-Java-1.7.0_80-Python-2.7.11, Pango-1.39.0-foss-2016a, cairo-1.14.6-foss-2016a, fontconfig-2.11.94-foss-2016a, freeglut-3.0.0-foss-2016a, gnuplot-5.0.3-foss-2016a, libGLU-9.0.0-foss-2016a, libQGLViewer-2.6.3-foss-2016a, libXfont-1.5.1-foss-2016a, libXft-2.3.2-foss-2016a, libgd-2.1.1-foss-2016a, matplotlib-1.5.1-foss-2016a-Python-2.7.11, matplotlib-1.5.1-foss-2016a-Python-3.5.1, pyGIMLi-20160803-foss-2016a-Python-2.7.11, pyGIMLi-20160803-foss-2016a-Python-3.5.1, scikit-image-0.12.3-foss-2016a-Python-2.7.11, scikit-learn-0.17.1-foss-2016a-Python-2.7.11, scikit-learn-0.17.1-foss-2016a-Python-3.5.1
	freetype-2.6.3-foss-2016a as dep of: CGAL-4.8-foss-2016a-Python-2.7.11, GConf-3.2.6-foss-2016a, GL2PS-1.3.9-foss-2016a-Mesa-11.2.1, GObject-Introspection-1.48.0-foss-2016a, GTK+-2.24.30-foss-2016a, HarfBuzz-1.2.7-foss-2016a, Mesa-11.2.1-foss-2016a, OpenCV-3.1.0-foss-2016a, Pango-1.40.1-foss-2016a, ParaView-4.4.0-foss-2016a, Pillow-3.2.0-foss-2016a-Python-2.7.11, PyQt5-5.7-foss-2016a-Python-2.7.11, Qt-4.8.7-foss-2016a-GLib-2.48.0, Qt5-5.6.0-foss-2016a, Qt5-5.7.0-foss-2016a, VTK-6.3.0-foss-2016a-Python-2.7.11, Yade-1.20.0-foss-2016a-Python-2.7.11, cairo-1.14.6-foss-2016a-GLib-2.48.0, fontconfig-2.11.95-foss-2016a, freeglut-3.0.0-foss-2016a-Mesa-11.2.1, grace-5.1.25-foss-2016a, libGLU-9.0.0-foss-2016a-Mesa-11.2.1, libQGLViewer-2.6.3-foss-2016a-Mesa-11.2.1, libXfont-1.5.1-foss-2016a-freetype-2.6.3, matplotlib-1.5.1-foss-2016a-Python-2.7.11-freetype-2.6.3, motif-2.3.5-foss-2016a, scikit-image-0.12.3-foss-2016a-Python-2.7.11, wxPython-3.0.2.0-foss-2016a-Python-2.7.11
```